HeterogeneousCore/TrivialSerialisation/test/test_catch2_main.cc

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
I see both this file and `test_catch2_SerialisationPluginFactory.cc` define `CATCH_CONFIG_MAIN`. Do you want to keep this file (and remove the definition from the other`, or remove this file?

Reply:
Thanks for the catch (🤣), it's a leftover from shuffling the tests around. I fixed `test_catch2_SerialisationPluginFactory.cc`.